src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdLifetime.cs

```diff
@@ -19,6 +22,13 @@ public partial class SystemdLifetime : IHostLifetime, IDisposable
         private CancellationTokenRegistration _applicationStartedRegistration;
         private CancellationTokenRegistration _applicationStoppingRegistration;
 
+        /// <summary>
+        /// Instantiates a <see cref="SystemdLifetime"/> object.
+        /// </summary>
+        /// <param name="environment">Contains information about the host.</param>
+        /// <param name="applicationLifetime">The <see cref="IHostApplicationLifetime"/> that tracks the service lifetime.</param>
```

Comment:
I think it's okay to add the type, but in general, you don't need to/shouldn't. It's shown automatically by IntelliSense and docs.